### PR TITLE
Fix it's typos, capitalize acronyms, rephrase

### PR DESCRIPTION
**Pull request by skierpage**
"It's" can only be an abbreviation of "it is" or "it has". "Its" is already the possessive pronoun, just as you don't write "I like hi's code."

### CODE REVIEW

**Comment by redwarp** [comment]
Oh, I did not expect a PR on the blog, though that is what I get from open sourcing it :-P
I should probably add a code of conduct, readme, license and so on then.

Damn it I can't believe I still make mistake with `it's` and `its`! And I've been speaking english for almost 20 years by now, I should know better!

So thanks for the typo fixes! I'm less enthusiastic with the rephrasing, but your mark a point. The blog post does a bit more than giving a few pointers, it's a step by step tutorial. So I'll accept it!

Cheers!

File: content/image-filters/index.md [hunk 1]
```diff
@@ -327,7 +327,7 @@ This method makes sure that there will be enough workgroup to cover each pixels.
 
 > If we had a width of 20 pixels and height 16, using the workgroup of dimension 16 by 16, we would be missing a band of 4 pixels by only creating a single workgroup. We would need to create a second workgroup to handle the extra pixels, and we would then be able to cover 32 pixels in width.
 >
-> Some work will go to waste, but it's better than not applying our filters to a band of pixels.
+> Some work will go to waste, but its better than not applying our filters to a band of pixels.
```
Comment (redwarp) [inline content/image-filters/index.md:330]: Actually just noticed that this was correct! I'll revert that, but write down `it is` instead.

Comment (skierpage) [reply content/image-filters/index.md:330]: Sorry. Whenever you corrects someone's grammar, you introduce your own misteaks. It's the law! :wink:

Comment (redwarp) [reply content/image-filters/index.md:330]: misteaks, I see what you did there ;-)

**Pull request merged**